### DISCUSSION TITLE
Can anyone help to recover access to my github account ???

### DISCUSSION THREAD

**Discussion by dashingwu** [community/community#102827] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Just for last hopeless trial for my old github account ...

I have not use github for about 4 years since left last company which contributed a lot to opensource community, and I didn't save the github-recovery-codes.txt and changed mobile phone.... Although I have the github related mailbox and exact password, I can't pass the 2FA to login ....

I can provide the all the github emails and PR records, and even ownership proof from other github account owners co-worked with me 4 years ago. And then, could anybody help me restore the access to my old github account?

Thanks!

**Comment by phuccoder2000** [comment, marked as answer]
Hello @dashingwu I think you should contact GitHub Support:
Start by reaching out to GitHub Support. Visit the GitHub Support page (https://support.github.com/contact) and submit a request, explaining your situation. Provide as much detail as possible, including the GitHub username, the associated email addresses, and any other relevant information. GitHub Support may guide you through the recovery process.

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @dashingwu! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

With that being said, the advice that @phuccoder2000 shared here is correct. The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by dashingwu** [comment]
Hi @ghostinhershell @phuccoder2000 

Thanks for response and sorry for posting in inproper category.
And according to your advice, I opened a tickets about 5 days ago, 
https://support.github.com/ticket/personal/0/2566057
but there is no response, is there any other channel to solve my github account issue?

Thanks again!
